src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportStatusBauService.cs

```diff
@@ -48,11 +49,30 @@ public class ImportStatusBauService : IImportStatusBauService
 
                     return queryResults.Where(result =>
                     {
-                        if (result.Message == "")
+                        try
+                        {
+                            var m = JsonConvert.DeserializeObject<ImportMessage>(result.Message);
+
+                            if (m.Release?.Id == null || m.Release?.Publication?.Id == null || m.MetaFileName == null)
```

Comment:
Do you know how do these end up being null in the first place?

Reply:
It normally happens when `result.Message == ""`, but I've done the above so that hopefully this catches errors where result.Message is otherwise correct, but not for the variables I want.

Reply:
I would suggest testing it with a proper message, but with one of the properties nulled out to see what exception is actually generated. I would hope that it should be easy enough to identify from the message without needing to test on specific variables.

I'd also suggest handling `result.Message == ""` before the serialization as there's no need to do any further work if it's empty.